server/util/path/index.js

```diff
@@ -110,6 +110,10 @@ function getExecutablePath(instance_type) {
  */
 function getMediaPaths() {
   var media_paths = [];
+  // User set media_path is first priority
+  if (media_path) {
```

Comment:
Thanks for your contribution.

I think media_path is never undefined / null:
```javascript
const media_path = (process.env.MEDIA_PATH) ? process.env.MEDIA_PATH : path.join(standardPaths.appDataLocation[0], 'media');
```

If the MEDIA_PATH env var wasn't set, standardPaths.appDataLocation[0] would be added twice.

You may replace it with:  `if (process.env.MEDIA_PATH) {`